check_sites.py

```diff
@@ -18,33 +18,48 @@
 import os
 from publicsuffix2 import PublicSuffixList
 
+def find_diff_sets(old_sets, new_sets):
```

Comment:
This doesn't find sets that were just modified by the PR, right? It only finds the sets that are completely new or completely removed.

Reply:
You're right. I had meant to use `difference = set(old_sets.items())^set(new_sets.items())`, which should give the correct answer, but is evidently not possible due to FpsSet being unhashable. I'll implement your suggestion below.

Reply:
Okay, taken care of now